docs/how-to-use-and-FAQ/quantized-int8-inference.md

```diff
@@ -23,7 +23,7 @@ find images/ -type f > imagelist.txt
 ./ncnn2table mobilenet-opt.param mobilenet-opt.bin imagelist.txt mobilenet.table mean=[104,117,123] norm=[0.017,0.017,0.017] shape=[224,224,3] pixel=BGR thread=8 method=kl
 ```
 
-* mean and norm are the values you passed to ```Mat::substract_mean_normalize()```
+* mean and norm are the values you passed to ```Mat::subtract_mean_normalize()```
```

Comment:
set subtract_mean_normalize as an exception ?

Reply:
ok, keep the old.